aws/app/ecs.tf

```diff
@@ -43,6 +43,8 @@ data "template_file" "form_viewer_task" {
     email_address_support           = var.email_address_support
     reprocess_submission_queue      = var.sqs_reprocess_submission_queue_id
     audit_log_queue_url             = var.sqs_audit_log_queue_id
+    zitadel_provider                = var.zitadel_provider_secret_arn
+    zitadel_administration_key      = var.zitadel_administration_key_secret_arn
```

Comment:
You'll also need to update the ECS task's IAM policy to allow it to read these new secret values:
https://github.com/cds-snc/forms-terraform/blob/ae1920ed6544ef0744350a077e3b3c1a59fa4bd0/aws/app/ecs_iam.tf#L27-L44